main.go

```diff
@@ -234,8 +237,10 @@ func openBrowser(url string) {
 		openCmd = exec.Command("chromium", chromeOptions(url)...)
 	case commandExists("chromium-browser"):
 		openCmd = exec.Command("chromium-browser", chromeOptions(url)...)
-	case pathExists("/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"):
-		openCmd = exec.Command("/Applications/Google Chrome.app/Contents/MacOS/Google Chrome", chromeOptions(url)...)
+	case pathExists(macPath):
+		openCmd = exec.Command(macPath, chromeOptions(url)...)
+	case pathExists(wslPath):
```

Comment:
I don't see the benefit to moving the constant declarations away?

Reply:
I didn't want write out the full path twice. I adjusted the mac path for consistency. I'll change it if you'd prefer it the original way.

Reply:
Ahh, I misread the code. The variables should be declared in a const block then this is good to go.

Reply:
Variables now declared as `const`